vitest.config.ts

```diff
@@ -0,0 +1,8 @@
+/// <reference types="vitest" />
+import { getViteConfig } from 'astro/config';
```

Comment:
This causes the process to hang after execution:

```
 Test Files  1 passed (1)
      Tests  6 passed (6)
   Duration  499ms (transform 51ms, setup 0ms, collect 62ms, tests 7ms, environment 0ms, prepare 133ms)

close timed out after 1000ms
Tests closed successfully but something prevents Vite server from exiting
You can try to identify the cause by enabling "hanging-process" reporter. See https://vitest.dev/config/#reporters
```

Hanging reporter doesn't help much:
```
There are 12 handle(s) keeping the process running

# Tinypool
file:///.../head-start/node_modules/tinypool/dist/esm/index.js:37              
file:///.../head-start/node_modules/tinypool/dist/esm/index.js:58              
file:///.../head-start/node_modules/tinypool/dist/esm/index.js:952             
file:///.../head-start/node_modules/vitest/dist/vendor/cli-api.E07AF1Yq.js:8192
file:///.../head-start/node_modules/vitest/dist/vendor/cli-api.E07AF1Yq.js:8845
file:///.../head-start/node_modules/vitest/dist/vendor/cli-api.E07AF1Yq.js:8870

# FILEHANDLE
(unknown stack trace)

# FILEHANDLE
(unknown stack trace)

...
```

Changing `{ getViteConfig } from 'astro/config'` to `{ defineConfig } from 'vitest/config'` solves the hanging issue. But not sure what new problems it creates.

We could for now also limit the `teardownTimeout`:

```ts
export default defineConfig({
  test: {
    teardownTimeout: 1000,
    // Vitest configuration options
  },
});
```

Reply:
For now I will use `defineConfig` as we are still in the early stages and there seem to be no additional problems so far. If we run into problems down the road, we can look into the other options, but for now this feels like a 'safe enough' stage to still make changes like this.